src/Microsoft.Azure.WebJobs.Host/Extensions/IConfigurationExtensions.cs

```diff
@@ -69,7 +69,13 @@ public static string GetPrefixedConnectionStringName(string connectionStringName
             return Constants.WebJobsConfigurationSectionName + connectionStringName;
         }
 
-        private static string FindConnectionString(IConfiguration configuration, string connectionName) =>
+        /// <summary>
+        /// Looks for a connection string by first checking the ConfigurationStrings section, and then the root.
+        /// </summary>
+        /// <param name="configuration">The configuration.</param>
+        /// <param name="connectionName">The connection string key.</param>
+        /// <returns></returns>
+        public static string FindConnectionString(this IConfiguration configuration, string connectionName) =>
```

Comment:
I exposed this helper publicly as I was using this pattern in several places.

Reply:
nit: The name wasn't super clear to me at first, I don't have a great suggestion, but can we name this something more obvious like `GetConnectionStringOrSetting` ?

Reply:
Done. Used `GetConnectionStringOrSetting`.